rebasehelper/build_tools/mock_tool.py

```diff
@@ -70,8 +80,24 @@ def _build_rpm(cls, srpm, results_dir, root=None, arch=None, builder_options=Non
 
         ret = ProcessHelper.run_subprocess(cmd, output=output)
 
+        tmp_build_log_path = os.path.join(results_dir, 'build.log')
+        tmp_mock_log_path = os.path.join(results_dir, 'mock_output.log')
+
+        build_log_path = os.path.join(rpm_results_dir, 'build.log')
+        mock_log_path = os.path.join(rpm_results_dir, 'mock_output.log')
+        root_log_path = os.path.join(rpm_results_dir, 'root.log')
+
         if ret != 0:
```

Comment:
This could be simplified and unnecessary indentation removed, e.g. like this:
```python
if ret == 0:
    return [f for f in PathHelper.find_all_files(results_dir, '*.rpm') if not f.endswith('.src.rpm')]

if ret == 1:
...
```